package.json

```diff
@@ -1,32 +1,23 @@
 {
   "name": "multicodec",
-  "version": "3.2.1",
+  "version": "4.0.0",
   "description": "JavaScript implementation of the multicodec specification",
-  "leadMaintainer": "Henrique Dias <hacdias@gmail.com>",
   "main": "src/index.js",
```

Comment:
I think this line needs to be changed to `dist/src/index.js` ?

Also probably nice to leave the `"types"` field.

Also worth considering or investigating whether aegir will build ESM or commonjs by default and ensuring that the desired module type is output.

Reply:
"types"